assets/js/Containers/FrevaGPT/index.js

```diff
@@ -234,44 +307,10 @@ class FrevaGPT extends React.Component {
           <div className="d-flex justify-content-between">
             <h2 onClick={this.toggleBotSelect}>FrevaGPT</h2>
           </div>
-  
-          <Col md={4}>
-            <Form.Select 
-              value={this.botModel}
-              onChange={(x) => { this.setState({ botModel: x.target.value }); }}
-              className="me-1 mb-3"
-              placeholder="Choose Chatbot"
-              hidden={this.state.hideBotModelList}>
-              {this.state.botModelList.map((x) => {
-                  return <option key={x}>{x}</option>;
-              })}
-            </Form.Select>
-            <SidePanel/>
-          </Col>
-  
-          <Col md={8}>
 
-            <ChatBlock></ChatBlock>
-            
-            {this.state.loading ? (<Row className="mb-3"><Col md={1}><Spinner/></Col></Row>) : null}
+          {this.state.botOkay === undefined ?? (<Spinner></Spinner>)}
+          {this.state.botOkay ? this.renderBotContent() : this.renderAlert() }
```

Comment:
`renderAlert()` gets also triggered if `botOkay` is undefined. Therefore, during startup and before `ping` is fully executed, the user will see an error-Message. You need to make sure, that the spinner and those two states are not visible at the same time.